RecoPixelVertexing/PixelVertexFinding/src/PVClusterComparer.cc

```diff
@@ -78,7 +78,7 @@ double PVClusterComparer::pTSquaredSum(const reco::Vertex &v) {
 void PVClusterComparer::setChisquareQuantile() {
   std::vector<double> maxChi2(20, 0.);
```

Comment:
please do not use variable that differs only for a "_"!

Reply:
are you sure 20 is safe?

Reply:
and, btw, why 20?

Reply:
I am not clear on why it was initialized with 20 elements.

there another function ( [PVClusterComparer::updateChisquareQuantile](https://github.com/cms-sw/cmssw/blob/41abcc72a7c65fa8d37e1d2b57f882a82f8b5f3b/RecoPixelVertexing/PixelVertexFinding/src/PVClusterComparer.cc#L88) ) that updates  the maxChi2_ vector if elements with ndof more than 20 are required.

Reply:
and its called when those elements need to be accessed, as seen [here](https://github.com/cms-sw/cmssw/blob/41abcc72a7c65fa8d37e1d2b57f882a82f8b5f3b/RecoPixelVertexing/PixelVertexFinding/src/PVClusterComparer.cc#L35#L36 ), so it should be safe.